core/src/appleMain/kotlin/ScannerBuilder.kt

```diff
@@ -1,11 +1,10 @@
 package com.juul.kable
 
-import com.benasher44.uuid.Uuid
 import com.juul.kable.logs.Logging
 import com.juul.kable.logs.LoggingBuilder
 
 public actual class ScannerBuilder {
-    public actual var services: List<Uuid>? = null
+    public actual var filters: List<Filter>? = null
```

Comment:
Same as above (a deprecation would be good for the old property).

```suggestion
    @Deprecated(message = "Replaced by filters property")
    public var services: List<Uuid>?
        set(value) {
            filters = value?.map { Filter.Service(it) }
        }
        get() = filters?.filterIsInstance<Filter.Service>()?.map { it.uuid }

    public actual var filters: List<Filter>? = null
```